doc/user/bgp.rst

```diff
@@ -228,6 +214,122 @@ internal or external.
    not enabled *bgpd* can't get interface information so `router-id` is set to
    0.0.0.0. So please set router-id by hand.
 
+
+.. _bgp-multiple-autonomous-systems:
+
+Multiple Autonomous Systems
+---------------------------
+
+FRR's BGP implementation is capable of running multiple autonomous systems at
+once. Each configured AS corresponds to a :ref:`zebra-vrf`. In the past, to get
+the same functionality the network administrator had to run a new *bgpd*
+process; using VRFs allows multiple autonomous systems to be handled in a
+single process.
+
+When using multiple autonomous systems, all router config blocks after the
+first one must specify a VRF to be the target of BGP's route selection. This
+VRF must be unique within respect to all other VRFs being used for the same
+purpose, i.e. two different autonomous systems cannot use the same VRF.
+However, the same AS can be used with different VRFs.
+
+.. note::
+
+   The separated nature of VRFs makes it possible to peer a single *bgpd*
+   process to itself, on one machine. This can be used for prototyping and
```

Comment:
The separated nature of VFs makes it possible to perform bgp peering through vrfs , within a single bgpd process, to itself. This can be used for prototyping*

Reply:
That's the exact same thing as what it says

Reply:
I believe Philippe's point here is that a bgp VRF may exists (and be useful, e.g. as a route reflector or route server) without a zebra VRF. -- which is correct.

Reply:
The "this can be used for prototyping": no need to mention it. Real use cases can be done by peering only BGP VRF instances between each other. By using only bgp vrf instances, one can have intereesting scenarii.

Reply:
Please check the new wording and make sure it meets your desires. If not, please provide me an exact text of what you want added.